github/Repository.py

```diff
@@ -1617,7 +1677,7 @@ def create_variable(self, variable_name: str, value: str) -> bool:
         status, headers, data = self._requester.requestJson(
             "POST", f"{self.url}/actions/variables", input=post_parameters
         )
-        return status == 201
+        return github.Variable.Variable(self._requester, headers, data, completed=True)
 
     def update_variable(self, variable_name: str, value: str) -> bool:
```

Comment:
There should also be an `edit` method in class `Variable` and `Secret`. Then it is arguable to remove this (reduces footprint of Repository class).